cmd/package.go

```diff
@@ -375,3 +382,72 @@ func NewPackageReportCmd() *cobra.Command {
 	packageReportCmd.Flags().String("name", "", "the package name")
 	return packageReportCmd
 }
+
+func NewPackageExportCmd() (*cobra.Command, error) {
+	packageExportCmd := &cobra.Command{
+		Use:   "export",
+		Short: "export packages",
+		Long:  `Export packages.`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			formatOption, err := cmd.Flags().GetString("format")
+			if err != nil {
+				return err
+			}
+			if formatOption != "json" && formatOption != "yaml" {
+				return fmt.Errorf("format must be json or yaml")
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := generateCobblerClient()
+			if err != nil {
+				return err
+			}
+
+			name, err := cmd.Flags().GetString("name")
+			if err != nil {
+				return err
+			}
+			formatOption, err := cmd.Flags().GetString("format")
+			if err != nil {
+				return err
+			}
+
+			itemNames := make([]string, 0)
+			if name == "" {
+				itemNames, err = Client.ListPackageNames()
+				if err != nil {
+					return err
+				}
+			} else {
+				itemNames = append(itemNames, name)
+			}
+
+			for _, itemName := range itemNames {
+				distro, err := Client.GetPackage(itemName, false, false)
```

Comment:
Should be `package`, not `distro`

Reply:
This would be a reserved name. I will instead use `linuxpackage`.